MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include requirements.txt
+recursive-include magstock *
```

Comment:
This file will be relevant if/when we start building RPMs and Debian packages, but for now is unused.